build-system/tasks/storybook/index.js

```diff
@@ -16,48 +16,82 @@
 'use strict';
 
 const argv = require('minimist')(process.argv.slice(2));
+const log = require('fancy-log');
 const {createCtrlcHandler} = require('../../common/ctrlcHandler');
 const {defaultTask: runAmpDevBuildServer} = require('../default-task');
 const {execScriptAsync} = require('../../common/exec');
 const {installPackages} = require('../../common/utils');
+const {yellow} = require('ansi-colors');
 
 const ENV_PORTS = {
   amp: 9001,
   preact: 9002,
 };
 
 /**
+ * @param {string} bin
  * @param {string} env 'amp' or 'preact'
+ * @param {...string} args
  * @return {!ChildProcess}
  */
-function launchEnv(env) {
-  const {ci, 'storybook_port': storybookPort = ENV_PORTS[env]} = argv;
-  return execScriptAsync(
-    [
-      './node_modules/.bin/start-storybook',
-      '--quiet',
-      '-s ../../../',
-      `-c ./${env}-env`,
-      `-p ${storybookPort}`,
-      ci ? '--ci' : '',
-    ].join(' '),
+const execStorybookScriptAsync = (bin, env, ...args) =>
+  execScriptAsync(
+    `./node_modules/.bin/${bin} --config-dir ./${env}-env ${args.join(' ')}`,
     {
       stdio: [null, process.stdout, process.stderr],
       cwd: __dirname,
       env: process.env,
     }
   );
+
+/**
+ * @param {string} env 'amp' or 'preact'
+ * @return {!ChildProcess}
+ */
+function launchEnv(env) {
+  const {ci, 'storybook_port': storybookPort = ENV_PORTS[env]} = argv;
+  return execStorybookScriptAsync(
+    'start-storybook',
+    env,
+    '--quiet',
```

Comment:
I'm ok with the info logs, but can we hide the warnings here? https://travis-ci.com/github/ampproject/amphtml/jobs/422916229#L835